ozaria/schemas/selectors/cinematic.js

```diff
@@ -0,0 +1,417 @@
+import { CAMERA_DEFAULT } from '../../lib/cinematic/CameraSystem'
```

Comment:
Is this file different from `/ozaria/schema/models/selectors/cinematic` ?

Reply:
This was my error. I got confused about where we should keep files related to the schemas. Do we want to also split out new schemas into `ozaria`? Or should we keep them all in the legacy folder? Because the server depends on them I think I'm for keeping them in the legacy folder instead  of spreading that server dependency.

Really interested in other opinions.

Reply:
I'm +1 for keeping cinematic schema in schemas folder.  I think the schemas folder will be around for a while and I think cinematics can be considered a top level system that could (theoretically) span other games / pieces of the code base.

Reply:
Schemas in legacy folder seems fine.